lib/matplotlib/widgets.py

```diff
@@ -1303,7 +1303,7 @@ def _release(self, event):
 
     def onmove(self, event):
         """Cursor move event handler and validator"""
-        if not self.ignore(event) and self.eventpress:
+        if not self.ignore(event):
```

Comment:
Not sure how I feel about shifting the eventpress test from onmove to other places. Could there be 3rd party code that depended upon that check being there?